From 317802fc2798a4bb1a53087d39123dee26cd8a3c Mon Sep 17 00:00:00 2001
From: Michael Forney <mforney@mforney.org>
Date: Fri, 23 Dec 2016 15:13:31 -0800
Subject: [PATCH] framebuffer: Set default backend to wld

---
 frontends/framebuffer/gui.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/netsurf/frontends/framebuffer/gui.c b/netsurf/frontends/framebuffer/gui.c
index 4d4c7334f..f191798d7 100644
--- a/netsurf/frontends/framebuffer/gui.c
+++ b/netsurf/frontends/framebuffer/gui.c
@@ -466,7 +466,7 @@ process_cmdline(int argc, char** argv)
 
 	LOG("argc %d, argv %p", argc, argv);
 
-	fename = "sdl";
+	fename = "wld";
 	febpp = 32;
 
 	fewidth = nsoption_int(window_width);
-- 
2.11.0

